__device-tests__/gutenberg-editor-contact-info.test.js

```diff
@@ -28,16 +34,218 @@ describe( 'Gutenberg Editor Contact Info Block tests', () => {
 		jasmine.getEnv().addReporter( reporter );
 	}
 
+	async function toggleSettings( drv, ePage, bk ) {
+		const buttonElementName = isAndroid()
+			? '//*'
+			: '//XCUIElementTypeButton';
+		const blockActionsMenuButtonLocator = `${ buttonElementName }[contains(@${ ePage.accessibilityIdXPathAttrib }, "Open Settings")]`;
+
+		if ( isAndroid() ) {
+			let checkList = await drv.elementsByXPath(
+				blockActionsMenuButtonLocator
+			);
+			while ( checkList.length === 0 ) {
+				await swipeUp( drv, bk ); // Swipe up to show remove icon at the bottom
+				checkList = await drv.elementsByXPath(
+					blockActionsMenuButtonLocator
+				);
+			}
+		}
+
+		const blockActionsMenuButton = await drv.elementByXPath(
+			blockActionsMenuButtonLocator
+		);
+		await blockActionsMenuButton.click();
+	}
+
+	async function findElement( drv, ePage, name, iosType ) {
+		const elementName = isAndroid()
+			? '//*'
+			: `//XCUIElementType${ iosType }`;
+		const blockLocator = `${ elementName }[contains(@${ ePage.accessibilityIdXPathAttrib }, "${ name }")]`;
+		const elements = await drv.elementsByXPath( blockLocator );
+		return elements[ 0 ];
+	}
+
+	async function dismissActionSheet() {
+		const action = await new wd.TouchAction( driver );
+		action.press( { x: 100, y: 100 } );
+		action.release();
+		await action.perform();
+	}
+
 	beforeAll( async () => {
 		driver = await setupDriver();
 		editorPage = new EditorPage( driver );
 	} );
 
+	beforeEach( async () => {
+		await editorPage.addNewBlock( contactInfoBlockName );
+	} );
+
 	it( 'should be able to see visual editor', async () => {
 		await expect( editorPage.getBlockList() ).resolves.toBe( true );
 	} );
 
-	//TODO: Add tests
+	it( 'should be able to add contact info block', async () => {
```

Comment:
NOTE: These tests are commented out for now because they are intermittently failing on Android. On Android, getLocation seems to be getting the wrong location coordinates for the block given, causing the top bar to be pressed instead of what should actually be pressed (which is the outside of the inner blocks in Contact Info, causing an inner block to be removed instead of the entire Contact Info block). The other difference causing this issue in Android but not in iOS is that editorPage.dismissKeyboard() deselects the child block in iOS but not in Android.